src/Directory.Build.props

```diff
@@ -13,6 +13,10 @@
     <CodeAnalysisRuleset>$(MSBuildThisFileDirectory)CodeAnalysis.ruleset</CodeAnalysisRuleset>
   </PropertyGroup>
 
+  <ItemGroup Condition="'$(IsTestProject)' == 'true' AND '$(IsTestSupportProject)' != 'true'">
```

Comment:
Nit: you may want a property to disable this, though I guess folks can always add such a property.

What’s the VS experience around this when loading an SLN after running only ‘build.cmd’?

Reply:
```
3>CSC : error CS0006: Metadata file 'C:\git\corefx2\artifacts\bin\CoreFx.Private.TestUtilities\netcoreapp-Debug\CoreFx.Private.TestUtilities.dll' could not be found
3>Done building project "System.Text.RegularExpressions.Tests.csproj" -- FAILED.
```

😥😥😥